lib/manageiq/providers/workflows/engine.rb

```diff
@@ -49,7 +49,13 @@ def self.floe_docker_runner
             )
           elsif MiqEnvironment::Command.is_appliance? || MiqEnvironment::Command.supports_command?("podman")
             options = {}
-            options["network"] = "host" unless Rails.env.production?
+            if Rails.env.production?
+              options["root"]        = Rails.root.join("data/containers/storage").to_s
+              options["storage-opt"] = "ignore_chown_errors=true" # Fix issues with pulling images as a non-root user
```

Comment:
NOTE I want to investigate this one, https://www.redhat.com/sysadmin/rootless-podman mentions adding an option to `/etc/subuid` and `/etc/subgid` that is the better way of doing this

Reply:
https://github.com/ManageIQ/manageiq-rpm_build/pull/415